app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -368,6 +368,26 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
     }))
   }
 
+  async function getStakingInfo(accountPubkeyHex) {
+    // TODO: not pubkey, address
+    const url = `${
+      ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL
+    }/api/account/info/${accountPubkeyHex}`
+    const response = await request(url)
+    return response
+  }
+
+  async function getValidStakepools() {
+    const url = `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/v2/stakePools`
+    const validStakepools = await request(url)
+
+    return {validStakepools}
```

Comment:
I would expect this to return directly the array of valid stakePools, the functions that use it also appear to just pluck that "validStakePools" property out of the returned object so seems like an unnecessary overhead

Reply:
Similarly, this previously also returned something other than the valid pools and was removed some time ago, I think this might change again but until then I ll fix it so we return it right away.